pyorbital/astronomy.py

```diff
@@ -39,55 +40,97 @@
 import datetime
 
 import numpy as np
+from typing import Tuple, Union
 
 from pyorbital import dt2np
 
-F = 1 / 298.257223563  # Earth flattening WGS-84
-A = 6378.137  # WGS84 Equatorial radius
+# Astronomical constants
+AU = 149597870700.0  # Astronomical unit in meters
+EARTH_ flattening = 1 / 298.257223563  # Earth flattening WGS-84
+EARTH_RADIUS = 6378.137  # WGS84 Equatorial radius in km
 MFACTOR = 7.292115E-5
+EARTH_SEMI_MAJOR_AXIS = 1.00000261 * AU  # Earth semi-major axis in meters
+EARTH_ECCENTRICITY = 0.01671123  # Earth orbital eccentricity
+EARTH_YEAR = 365.25636  # Length of Earth year in days
+PERIHELION_DAY = 3  # Day of year with Earth in perihelion (approx.)
 
+# Type alias for time arguments
+TimeLike = Union[dt.datetime, np.datetime64, np.ndarray]
 
-def jdays2000(utc_time):
+
+def jdays2000(utc_time: TimeLike) -> Union[float, np.ndarray]:
     """Get the days since year 2000.
+
+    :param utc_time: UTC time
+    :type utc_time: datetime.datetime, numpy.datetime64, or array-like
+    :return: Days since year 2000
+    :rtype: float or numpy.ndarray
```

Comment:
I'm not sure if pyorbital has a defined docstring format it follows, but in Satpy we've been using Google style docstrings:

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Would it be possible to reformat these to match? If other parts of pyorbital are using numpy style docstrings then I'd be open to that as well.

Reply:
Additionally, I think you can remove the type in the docstrings if you have the annotations. We'll need to possibly enable the correct sphinx options to copy the annotations to the rendered documentation, but that can come after everything else is sorted.